ttnn/cpp/ttnn/operations/experimental/dropout/dropout.hpp

```diff
@@ -0,0 +1,17 @@
+
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+namespace ttnn::operations::experimental {
+
+struct DropoutOperation {
+    static Tensor invoke(const Tensor& input_tensor, float prob, float scale, uint32_t seed);
+};
+}  // namespace ttnn::operations::experimental
+namespace ttnn::experimental {
+constexpr auto dropout =
```

Comment:
would be good to just say that `ttnn::experimental::dropout` is an alias to `ttnn::prim::dropout`

Reply:
do you mean add comment?

Reply:
need to discuss, not in scope of this PR.